index.js

```diff
@@ -88,9 +88,25 @@ class ServerlessPlugin {
             if (f.runtime === 'provided') {
                 throw new this.serverless.classes.Error(`Bref 1.0 layers are not compatible with the "provided" runtime.\nTo upgrade to Bref 1.0, you have to switch to "provided.al2" in serverless.yml for the function "${name}".\nMore details here: https://bref.sh/docs/news/01-bref-1.0.html#amazon-linux-2`);
             }
+
+            this.defineBrefFpmTimeout(name)
         }
     }
 
+    defineBrefFpmTimeout(name) {
+        const timeout = this.serverless.service.functions[name].timeout ??
+            this.serverless.service.provider?.environment?.timeout ??
+            29 // We assume API Gateway Limit.
```

Comment:
Shouldn't the default be 6 seconds (to be confirmed) , the default timeout for functions in the serverless framework?

If a timeout is not explicitly defined, sls would deploy it with a timeout of 6 seconds and fpm would timeout at 28 seconds. This would defeat the purpose of this PR.

Reply:
It's a tricky thing. If we go with 6 seconds here, anybody deploying without Serverless will suddenly see a huge timeout drop in their API Gateway. I know that if they're using API gateway they cannot have it bigger than 30 seconds. For ALB, the use case is much more limited and much more likely to be used by people aware of how AWS works.
Choosing a large amount because of API Gateway will cover the most common case and anybody having issues with 6 second timeout can always increase it to 30 seconds.